README.md

```diff
@@ -4,21 +4,402 @@ Complete the following queries and paste the sql results under each question. Us
+Rancho grande	null
+Spécialités du monde	null
+The Cracker Box	null
+Trail's Head Gourmet Provisioners	null
+
 4. Select the list of customer_ids, and the number or orders, sorted descending
+
+SELECT Customers.CustomerID, COUNT(Orders.OrderID) as [Total Count] FROM Customers
+LEFT JOIN Orders
+ON Orders.CustomerID = Customers.CustomerID
+Group By Customers.CustomerID
+ORDER BY [Total Count] Desc;
+
+Number of Records: 91
+CustomerID	Total Count
+20	10
+63	7
+65	7
+87	7
+37	6
+75	6
+41	5
+46	5
+51	5
+7	4
+10	4
+24	4
+25	4
+55	4
+61	4
+71	4
+80	4
+86	4
+5	3
+9	3
+21	3
+36	3
+38	3
+44	3
+49	3
+59	3
+60	3
+66	3
+69	3
+72	3
+4	2
+14	2
+17	2
+19	2
+28	2
+29	2
+34	2
+35	2
+39	2
+48	2
+58	2
+62	2
+67	2
+68	2
+73	2
+76	2
+81	2
+83	2
+84	2
+85	2
+88	2
+89	2
+2	1
+3	1
+8	1
+11	1
+13	1
+15	1
+16	1
+18	1
+23	1
+27	1
+30	1
+31	1
+33	1
+47	1
+52	1
+54	1
+56	1
+70	1
+77	1
+79	1
+90	1
+91	1
+1	0
+6	0
+12	0
+22	0
+26	0
+32	0
+40	0
+42	0
+43	0
+45	0
+50	0
+53	0
+57	0
+64	0
+74	0
+78	0
+82	0
+
 5. Select the SUM of all orders in 1997
+
+SELECT Count(OrderID) FROM [Orders]
+where OrderDate like '1997-%'
+
+Count(OrderID)
+44
+
 6. Select the name of the shipper, and the total number of orders shipped
+
+Select Shippers.ShipperName, COUNT(Orders.OrderID) as [Total Count]
+FROM Orders
+JOIN Shippers
+ON Shippers.ShipperId = Orders.ShipperID
+Group By ShipperName
+ORDER BY [Total Count] Desc;
+
+Number of Records: 3
+ShipperName	Total Count
+United Package	74
+Federal Shipping	68
+Speedy Express	54
+
 7. Select the ID and amount of the most expensive order
+
+SELECT ID, MAX(TotalPrice)
+FROM (
+SELECT
+Orders.OrderID as ID, SUM(OrderDetails.Quantity * Products.Price) as TotalPrice
+FROM Orders
+INNER JOIN OrderDetails
+ON Orders.OrderID=OrderDetails.OrderID
+INNER JOIN Products
+ON Products.ProductID = OrderDetails.ProductID
+Group by OrderDetails.OrderID
+ORDER BY TotalPrice Desc);
+
+Number of Records: 1
+ID	MAX(TotalPrice)
+10372	15353.6
+
+
 8. Select the names of the employees that have shipped Tofu
+
+We couldn't get this to work with concatenating first and last name. :(
+SELECT Employees.FirstName+' '+Employees.LastName AS Name
+FROM Employees
+JOIN Orders
+ON Employees.EmployeeID=Orders.EmployeeID
+JOIN OrderDetails
+ON Orders.OrderID=OrderDetails.OrderID
+JOIN Products
+ON Products.ProductID=OrderDetails.ProductID
+WHERE Products.ProductName="Tofu"
+
+We got it to work when we selected the first and last columns separately
+SELECT FirstName, LastName
+FROM Employees
+JOIN Orders
+ON Employees.EmployeeID=Orders.EmployeeID
+JOIN OrderDetails
+ON Orders.OrderID=OrderDetails.OrderID
+JOIN Products
+ON Products.ProductID=OrderDetails.ProductID
+WHERE Products.ProductName="Tofu"
+
+Number of Records: 8
+FirstName	LastName
+Michael	Suyama
+Nancy	Davolio
+Steven	Buchanan
+Janet	Leverling
+Nancy	Davolio
+Janet	Leverling
+Laura	Callahan
+Margaret	Peacock
+
 9. Select the name of the customer with the most expensive order
+
+SELECT Name, MAX(TotalPrice)
+FROM (
```

Comment:
Why do you have a subquery here? Looks like your'e just select the same columns?